agent/ecscni/plugin.go

```diff
@@ -111,18 +114,34 @@ func (client *cniClient) setupNS(cfg *Config) (*current.Result, error) {
 	}
 
 	seelog.Debugf("[ECSCNI] Starting ENI (%s) setup in the the container namespace: %s", cfg.ENIID, cfg.ContainerID)
-	os.Setenv("ECS_CNI_LOGLEVEL", logger.GetLevel())
-	defer os.Unsetenv("ECS_CNI_LOGLEVEL")
 
-	// Invoke eni plugin ADD command
-	result, err := client.add(runtimeConfig, cfg, client.createENINetworkConfig)
-	if err != nil {
-		return nil, errors.Wrap(err, "cni setup: invoke eni plugin failed")
+	// Invoke eni plugin ADD command based on the type of eni plugin
+	if cfg.ENIType == apieni.BranchENIType {
+		os.Setenv("VPC_CNI_LOG_LEVEL", logger.GetLevel())
```

Comment:
curious, why do we need this log level variable to be set?

Reply:
VPC CNI plugin uses this environment variable [here](https://github.com/aws/amazon-vpc-cni-plugins/blob/master/logger/logger.go#L24-L26).

Reply:
We can change the log config behavior to match the ecs-cni plugin if that is going to result in simpler agent code.